### PR TITLE
/feature-govukpage-test

### PR DESCRIPTION
**Pull request by SimRoberts9**
This will test that 10 strings can have values assigned to them and return this value, it is also testing 2 moving parts that if null they are assigned a default value set in GovukPage.

### CODE REVIEW

File: tests/Unit/Helpers/GovukPage/ConfirmTest.php [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+use AnthonyEdmonds\GovukLaravel\Helpers\GovukPage;
+use AnthonyEdmonds\GovukLaravel\Pages\Page;
+use AnthonyEdmonds\GovukLaravel\Tests\TestCase;
+
+class ConfirmTest extends TestCase
+{
+    protected array $page;
+
+    protected function makePage(
```
Comment (AnthonyEdmonds) [inline tests/Unit/Helpers/GovukPage/ConfirmTest.php:13]: Methods should be ordered by their visibility - public at the top, then protected, then private at the bottom.

File: tests/Unit/Helpers/GovukPage/ConfirmTest.php [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+        $this->page = GovukPage::confirm(
+            'good',
+            'for',
+            'apples',
+            'table',
+            'let',
+            'potato',
+            $otherButtonLabel,
+            $otherButtonHref,
+            'warning',
```
Comment (AnthonyEdmonds) [inline tests/Unit/Helpers/GovukPage/ConfirmTest.php:26]: This should refer to a const from Page in case the label changes.

File: tests/Unit/Helpers/GovukPage/ConfirmTest.php [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+            $this->page['otherButtonHref'],
+        );
+    }
+
+    public function testHasSubmitButtonType(): void
+    {
+        $this->makePage();
+
+        $this->assertEquals(
+            'warning',
```
Comment (AnthonyEdmonds) [inline tests/Unit/Helpers/GovukPage/ConfirmTest.php:116]: As above, this should reference an const from Page

File: tests/Unit/Helpers/GovukPage/ConfirmTest.php [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+    {
+        $this->makePage(null, null);
+
+        $this->assertEquals(
+            'let',
+            $this->page['otherButtonHref'],
+        );
+    }
+
+    public function testOtherButtonLabelIsTrueWhenNull(): void
```
Comment (AnthonyEdmonds) [inline tests/Unit/Helpers/GovukPage/ConfirmTest.php:141]: The test title doesn't quite match up with what is being asserted. This isn't true, it is being checked against a default value. Perhaps something along the lines of `testOtherButtonLabelIsDefaultWhenNull` then reference the default button label const, if there is one.

**Review by AnthonyEdmonds — CHANGES_REQUESTED** [review]

File: tests/Unit/Helpers/GovukPage/ConfirmationTest.php [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+use AnthonyEdmonds\GovukLaravel\Helpers\GovukPage;
+use AnthonyEdmonds\GovukLaravel\Pages\Page;
+use AnthonyEdmonds\GovukLaravel\Tests\TestCase;
+
+class ConfirmationTest extends TestCase
+{
+    protected array $page;
+
+    public function testCreatedTitle(): void
```
Comment (AnthonyEdmonds) [inline tests/Unit/Helpers/GovukPage/ConfirmationTest.php:13]: Did you mean sets or has title?

**Review by AnthonyEdmonds — CHANGES_REQUESTED** [review]

File: tests/Unit/Helpers/GovukPage/QuestionTest.php [hunk 6]
```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace AnthonyEdmonds\GovukLaravel\Tests\Unit\Helpers\GovukPage;
+
+use AnthonyEdmonds\GovukLaravel\Helpers\GovukPage;
+use AnthonyEdmonds\GovukLaravel\Pages\Page;
+use AnthonyEdmonds\GovukLaravel\Questions\Question;
+use AnthonyEdmonds\GovukLaravel\Tests\TestCase;
+
+class QuestionTest extends TestCase
```
Comment (AnthonyEdmonds) [inline tests/Unit/Helpers/GovukPage/QuestionTest.php:10]: This is missing a test that asserts that the page title has been set from the question.

**Review by AnthonyEdmonds — CHANGES_REQUESTED** [review]
